next.config.js

```diff
@@ -1,5 +1,8 @@
 const withSourceMaps = require('@zeit/next-source-maps')();
 const withBundleAnalyzer = require('@zeit/next-bundle-analyzer');
+const withMDX = require('@next/mdx')({
+  extension: /\.mdx?$/,
+});
```

Comment:
```suggestion
const withMDX = rquire('@next/mdx')({
  extension: /\.mdx$/,
});
```

let's require it be `mdx`

Reply:
fixed